docs/cli.md

```diff
@@ -129,13 +129,25 @@ Builds, (re)creates, starts, and attaches to containers for a service.
 
 Linked services will be started, unless they are already running.
 
-By default, `docker-compose up` will aggregate the output of each container and,
-when it exits, all containers will be stopped. Running `docker-compose up -d`,
-will start the containers in the background and leave them running.
+By default, `docker-compose up` will aggregate the output of each container
+and, when it exits, all containers will be stopped. Running
+`docker-compose up -d`, will start the containers in the background and leave
+them running.
 
```

Comment:
### up

Builds, (re)creates, starts, and attaches to containers for a service.

Unless they are already running, link services start.

By default, `docker-compose up` aggregates the output of each container. When it exits, all containers are stopped. Running
`docker-compose up -d`, starts the containers in the background and leaves
them running.

By default, if there are existing containers for a service, and the service's
configuration or image has changed since they were created, `docker-compose up`
stops and recreates them (preserving mounted volumes), so that the changes
are picked up.

If you do not want containers stopped and recreated, use `--no-recreate` to start any stopped containers.

If you want to force compose to stop and recreate all containers, use
the `--force-recreate` flag.